src/modelfree/common/policy_loader.py

```diff
@@ -22,7 +22,7 @@ def __init__(self, policy, vec_normalize):
         super().__init__(policy, policy.sess)
         self.vec_normalize = vec_normalize
 
-    def predict(self, observation, state=None, mask=None, deterministic=False):
+    def predict(self, observation, state=None, mask=None, deterministic=True):
```

Comment:
Why are we changing this to be the default? I think that's going to affect a lot of things (e.g. `score_agent`).